assets/src/stories-editor/blocks/amp-story-post-date/index.js

```diff
@@ -17,9 +17,8 @@ export const settings = {
 	category: 'common',
 	icon: 'clock',
 	keywords: [
-		__( 'post', 'amp' ),
+		__( 'story', 'amp' ),
 		__( 'publish date', 'amp' ),
-		__( 'published date', 'amp' ),
 		__( 'date', 'amp' ),
```

Comment:
^ This is also redundant since `publish date` already covers the `date` keyword.

Reply:
Makes sense!